ProcessOptimizer/space/space.py

```diff
@@ -313,6 +317,44 @@ def lhs_arange(self, n):
         # Transform the samples to the range used for this dimension
         return samples*(self.high - self.low) + self.low
 
+    def evenly_sample(self, num_points, border_width=None):
+        """Return `n_points` evenly spaced points from a Dimension.
+
+        Parameters
+        ----------
+        * `num_points` [int]
+            The number of points to sample.
+
+        * `border_width` [int]
+            How wide to make the borders. If None, no borders are returned.

```

Comment:
I don't understand why this number should be an int? Are the borders not generally a fraction of the dimension length? If not, then how are they defined? What happens when a dimension is, say, [0.0,  1.0]?